apps/include/apps.h

```diff
@@ -66,8 +66,8 @@ BIO *bio_open_owner(const char *filename, int format, int private);
 BIO *bio_open_default(const char *filename, char mode, int format);
 BIO *bio_open_default_quiet(const char *filename, char mode, int format);
 CONF *app_load_config_bio(BIO *in, const char *filename);
-#define app_load_config(filename) app_load_config_internal(filename, 0)
-#define app_load_config_quiet(filename) app_load_config_internal(filename, 1)
+# define app_load_config(filename) app_load_config_internal(filename, 0)
+# define app_load_config_quiet(filename) app_load_config_internal(filename, 1)
 CONF *app_load_config_internal(const char *filename, int quiet);
```

Comment:
Are all the changes in this file codestyle-related? Can't we don't introduce them or introduce them separately?

Reply:
The indentation if preprocessor directives according to the `#if` nesting level is part of the coding style and checked by the tool.
Before these changes, `check-format.pl` complained like this:
````
apps/include/apps.h:69:'#if' nesting indent = 0 != 1:#define app_load_config(filename) app_load_config_internal(filename, 0)
apps/include/apps.h:70:'#if' nesting indent = 0 != 1:#define app_load_config_quiet(filename) app_load_config_internal(filename, 1)
```